src/Components/Dropdowns/Composed/TypeAheadDropDown.tsx

```diff
@@ -94,28 +66,41 @@ export const TypeAheadDropDown: FC<OwnProps> = ({
 
   const [selectIndex, setSelectIndex] = useState(-1)
   const [queryResults, setQueryResults] = useState(items)
-  const [menuOpen, setMenuOpen] = useState<MenuStatus>(MenuStatus.Closed)
-
+  const [menuStatus, setMenuStatus] = useState<MenuStatus>(MenuStatus.Closed)
```

Comment:
`menuStatus` seemed like a better name, `menuOpen` sounds like a boolean when its not (?)

Reply:
LGTM.